mantidproject/mantid
# Add Python 3 packages debian developer package

Description of work.

Tidies up the control file and adds required Python 3 packages.

**To test:**

* Check the list
* On a Debian/Ubuntu machine switch to the  `buildconfig/dev-packages/deb/mantid-developer` directory and run `equivs-build ns-control` (you may need to install `equivs`).
* You can simulate installing the new package with `gdebi --apt-line mantid-developer_1.3.2_all.deb`

*No issue*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
